es/constants.js

```diff
@@ -1,2 +1,29 @@
 export var KEY_PREFIX = 'reduxPersist:';
-export var REHYDRATE = 'persist/REHYDRATE';
\ No newline at end of file
+export var REHYDRATE = 'persist/REHYDRATE';
+export var ENTITY_KEYS = [
+  "entities",
```

Comment:
Can we have this in the TR repo and then a function where we setEntityKeys?
This is going to be hard to maintain if we ever want to add another entity because we will need to add the key in this PR as well